quilt/tools/main.py

```diff
@@ -152,7 +154,10 @@ def check_hash(group, hashstr):
         func(**kwargs)
         return 0
     except command.CommandException as ex:
-        print(ex, file=sys.stderr)
+        if str(ex) == command.DEFAULT_QUILT_YML_NOT_FOUND_MSG:
```

Comment:
This is much safer than the previous version using regex, but it's still pretty hard to understand this code without knowing the background. IMHO, it would be easier to follow the code if all the logic for install were above where the install args are defined.